drizzlepac/hapmultisequencer.py

```diff
@@ -377,19 +377,22 @@ def run_mvm_processing(input_filename, skip_gaia_alignment=True, diagnostic_mode
 
             log.info("Preparing configuration parameter values for filter product {}".format(filter_item.drizzle_filename))
             filter_item.configobj_pars = config_utils.HapConfig(filter_item,
+                                                                hap_pipeline_name='mvm',
                                                                 log_level=log_level,
                                                                 use_defaults=use_defaults_configs,
                                                                 input_custom_pars_file=input_custom_pars_file,
                                                                 output_custom_pars_file=output_custom_pars_file)
 
             for edp in filter_item.edp_list:
                 edp.configobj_pars = config_utils.HapConfig(edp,
+                                                            hap_pipeline_name='mvm',
                                                             log_level=log_level,
                                                             use_defaults=use_defaults_configs,
                                                             input_custom_pars_file=input_custom_pars_file,
                                                             output_custom_pars_file=output_custom_pars_file)
         log.info("The configuration parameters have been read and applied to the drizzle objects.")
-
+        import pdb
+        pdb.set_trace()
```

Comment:
Please make sure these debugging statements are removed before this gets merged.

Reply:
Issue resolved by commit [e97e07b](https://github.com/spacetelescope/drizzlepac/pull/1287/commits/e97e07bab0d45cfb7a1b654890fb1def17135de7)